src/main/scala/org/scalameter/Persistor.scala

```diff
@@ -6,15 +6,15 @@ package org.scalameter
 
 
 trait Persistor {
-  def load(context: Context): History
-  def save(context: Context, h: History): Unit
+  def load[T: Manifest](context: Context): History[T]
+  def save[T](context: Context, h: History[T]): Unit
```

Comment:
I did not dive into the details, but: sounds like an alternative would be to embed the manifest information about `T` into the `History` class, so that when it's deserialized, Jackson knows how to select the right pickler for its values.

Would that work?